lab-geoff/karma.conf.js

```diff
@@ -0,0 +1,26 @@
+const webpackConfig = require('./webpack.config.js');
+webpackConfig.entry = {};
+
+module.exports = function(config) {
+  config.set({
+    webpack: webpackConfig,
+    basePath: '',
+    frameworks: ['jasmine'],
+    files: [
+      'test/*.js'
```

Comment:
Looks like you may want to expand this glob, as well as the preprocessors to look for both files and directories with more files inside of the test dir.